lib/modules/apostrophe-pages/index.js

```diff
@@ -291,6 +291,76 @@ module.exports = {
       ], callback);
     };
 
+    // Task to add a page from apostrophe
+    // command line
+
+    self.apos.tasks.add('apostrophe-pages', 'add',
+      'Usage: node app apostrophe-pages:add title /slug type /parentslug\n\n' +
+      'This adds a new page .\n',
+      function(apos, argv, callback) {
+        return self.addFromTask(callback);
+      }
+    );
+
+    self.addFromTask = function(callback) {
+      var argv = self.apos.argv;
+      if(argv._.length !== 5 ){
+        return callback('Incorrect number of arguments.');
+      }
+
+      var title = argv._[1];
+      var slug = argv._[2];
+      var type = argv._[3];
+      var parentslug = argv._[4];
+      var req = self.apos.tasks.getReq();
+
+      var page = {
+          editUsersIds: [],
+          loginRequired: '',
+          orphan: '0',
+          published: '1',
+          slug: slug,
+          tags: [],
+          title: title,
+          type: type
+        };
+
```

Comment:
It would make more sense to populate this by looping over the schema and populating the fields that exist as command line arguments (properties of argv).

In particular it's not a good idea to force `published=1`. If it's not the default then it's not the default (on that particular site), but you would be able to pass `published=1`.